test/SlowTests/Cluster/CompareExchangeTests.cs

```diff
@@ -0,0 +1,62 @@
+using System;
+using System.Linq;
+using System.Threading.Tasks;
+using FastTests;
+using Raven.Client.Util;
+using Raven.Server.ServerWide.Commands;
+using Sparrow.Json;
+using Sparrow.Json.Parsing;
+using Xunit;
+using Xunit.Abstractions;
+
+namespace SlowTests.Cluster;
+
+public class CompareExchangeTests : RavenTestBase
+{
+    public CompareExchangeTests(ITestOutputHelper output) : base(output)
+    {
+    }
+    
+    [Fact]
+    public async Task AddOrUpdateCompareExchangeCommand_WhenCommandSentTwice_SecondAttemptShouldNotReturnNull()
```

Comment:
this is stress test, but I do want it to run pretty regularly, so let's make a `NightlyBuildFact`

Reply:
done

By the way it tasks to run this test 13 sec in Release and 17 in Debug (on my PC)